<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaugeSetpointWidget</class>
 <widget class="QWidget" name="GaugeSetpointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>20</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>20</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>SRC01-VA-IMG1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblGauge">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):getPressure</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>prefix=SRC01-VA-IMG1</string>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELabel" name="lblSetpointON">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):getOnSetpoint$(sp)</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELineEdit" name="txtSetpointON">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):setOnSetpoint$(sp)</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELabel" name="lblSetpointOFF">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):getOffSetpoint$(sp)</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELineEdit" name="txtSetpointOFF">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):setOffSetpoint$(sp)</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELabel" name="lblRelay1State">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):getChannel$(sp)Relay1State</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblRelay2State">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-VA-GC$(gc):getChannel$(sp)Relay1State</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>gc=1,sp=1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
